ui/app/rif/components/searchDomains.js

```diff
@@ -16,7 +16,7 @@ class SearchDomains extends Component {
 
   _handleKeyDown = (e) => {
     if (e.key === 'Enter') {
-      const typedDomain = e.target.value.toLowerCase();
+      const typedDomain = (e.target.value.toLowerCase() && e.target.value.toLowerCase().indexOf('.rsk') === -1) ? (e.target.value.toLowerCase() + '.rsk') : e.target.value.toLowerCase();
```

Comment:
Maybe we can move this to utils.js or something better, what do you think about this?